Dockerfile-alpine.template

```diff
@@ -43,4 +43,20 @@ RUN adduser -D -u 1000 node \
     && rm -Rf "node-v$NODE_VERSION" \
     && rm "node-v$NODE_VERSION.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
 
+ENV YARN_VERSION 0.0.0
```

Comment:
Yeah, GitHub looks incorrect and 0.21.3 has been promoted to stable. cc @bestander